services/cosi-driver-nutanix/0.0.4/defaults/cm.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cosi-driver-nutanix-0.0.4-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |-
+    cosiController:
+      enabled: false # This should be deployed during k8s cluster creation by konvoy.
+    objectstorageProvisionerSidecar:
+      image:
+        registry: gcr.io
+        # keep this in sync with the sidecar that is deployed in CephCOSIDriver to avoid duplicate images in airgapped bundle.
+        repository: k8s-staging-sig-storage/objectstorage-sidecar
+        tag: v20250117-a29e5f6
+        pullPolicy: IfNotPresent
+    image:
+      registry: ghcr.io
+      repository: nutanix-cloud-native/cosi-driver-nutanix
+      # Overrides the image tag whose default is the chart appVersion.
+      tag: v0.0.4
+      pullPolicy: IfNotPresent
+    secret:
+      enabled: true
+      # Nutanix Object Store instance endpoint, eg. "http://10.51.142.82:80".
+      endpoint: "<TO BE REPLACED>"
+      # Admin IAM Access key to be used for Nutanix Objects.
+      access_key: "<TO BE REPLACED>"
+      # Admin IAM Secret key to be used for Nutanix Objects.
+      secret_key: "<TO BE REPLACED>"
+      # PC Credentials.
+      pc_ip: "<TO BE REPLACED>"
+      pc_port: "9440"
+      pc_username: admin
+      pc_password: "<TO BE REPLACED>"
```

Comment:
I think we should disable the secret due to AppDeployment overrides only being configurable via ConfigMap meaning credentials etc would be in ConfigMap and diagnostic bundles, etc.

I think we should use federated secrets for this instead.

Reply:
Done 2e1997c9ce3562a7106d8f323d5622532e6a7b1f